src/corefx/System.Globalization.Native/pal_calendarData.c

```diff
@@ -4,6 +4,7 @@
 
 #include <assert.h>
 #include <stdlib.h>
+#include <stdbool.h>
```

Comment:
This should go in as a separate PR....

Reply:
Sounds good, I was just checking to see if it fixed the mac failures (which it looks like it did). We can remove this commit.